### PR TITLE
Remove httpd image tag from mirrorbits

### PR DESCRIPTION
**Pull request by karthikmurali60**
### Description

The httpd docker image tag is already kept upto date in the mirrorbits helm chart

### CODE REVIEW

**Comment by karthikmurali60** [comment]
@lemeurherve I have created a new PR with the changes as discussed.

File: config/mirrorbits.yaml
Comment (lemeurherve) [inline config/mirrorbits.yaml:7]:
```suggestion
```
The `repository` is already set to the same `httpd@sha256` value in the mirrorbits chart at https://github.com/jenkins-infra/helm-charts/blob/main/charts/mirrorbits/values.yaml#L12-L15, no need to set it again.

**Review by lemeurherve — COMMENTED** [review]
Thanks for the contribution @karthikmurali60 🤗 

Small suggestion to remove the `repository` value then good to go!

**Comment by karthikmurali60** [comment]
> Thanks for the contribution @karthikmurali60 🤗
> 
> Small suggestion to remove the `repository` value then good to go!

Made the change in the latest commit

**Review by lemeurherve — APPROVED** [review]
The checks look good: https://github.com/jenkins-infra/kubernetes-management/pull/4226/checks?check_run_id=15686411097

I'll wait until tomorrow to merge this PR so I can monitor mirrorbits in case the resulting `httpd` update breaks something.

**Review by dduportal — APPROVED** [review]
diff looks good to me, nice job!

**Comment by karthikmurali60** [comment]
@lemeurherve can we merge this PR ?

**Comment by lemeurherve** [comment]
@karthikmurali60 merged, thanks again for your contribution!

**Pull request merged**
